app/javascript/components/concept-map/Concept.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { useEffect, useRef, useState, MouseEventHandler } from 'react'
 import { CompleteIcon } from './CompleteIcon'
+import * as Tooltips from '../tooltips'
```

Comment:
Should this just import `Tooltips.Concept` rather than everything?

Reply:
It could be, I just followed the already set pattern.

Reply:
Let's import the single thing. Could you change this when you get a min pls? :)